source/features/index.tsx

```diff
@@ -104,6 +106,11 @@ const globalReady: Promise<RGHOptions> = new Promise(async resolve => {
 	// Options defaults
 	const options = await optionsStorage.getAll();
 
+	// Fetch hotfixes asynchronously
+	const hotfix = await cache.get('hotfix');
+	await fetchHotfixesAsynchronously();
```

Comment:
The function is not meant to be awaited. 


```suggestion
	checkForHotfixes();
```

[cache.function](https://github.com/fregante/webext-storage-cache#cachefunctiongetter-options) is a shortcut that either gets the value from the cache (instant) or calls the passed function to generate the value (might take longer than 500ms), cache it, and return it.

Normally, you would use it as:

```js
const hotfix = await checkForHotfixes();
```

However, in this case we don't want the whole extension to have to await that function, so we just check the cache manually first (`cache.get` is nearly instant) and then we call `checkForHotfixes` so that the next time `cache.get` is called, the value will probably be cached.